pages/positions/consulting-software-engineer.md

```diff
@@ -27,19 +27,23 @@ supervisory_status: 'No'
 ---
 
 {% if page.state == 'upcoming' %}
-{{ page.org }} will soon be accepting applications for a GS-{{ page.gs_level }} {{ page.title }}.
-{% if page.opens == 'tbd' %} The target date for when this position will be officially open to application has not yet been determined. If you'd like to be notified when this position is open, sign up to our [mailing list](https://public.govdelivery.com/accounts/USGSATTS/subscriber/new?topic_id=USGSATTS_6). {% else %} Applications will be open for submission on {{ page.opens | date: '%l:%M%P %Z on %A, %B %e, %Y' }}. Check out [Join TTS Hiring Process]({{site.baseurl}}/hiring-process/) to learn more about the application process. 
+  {{ page.org }} will soon be accepting applications for a GS-{{ page.gs_level }} {{ page.title }}.
+  {% if page.opens == 'tbd' %} The target date for when this position will be officially open to application has not yet been determined. If you'd like to be notified when this position is open, sign up to our [mailing list](https://goo.gl/forms/QMbyPse8f4rfnZ9z2).
+  {% else %} Applications will be open for submission on {{ page.opens | date: '%l:%M%P %Z on %A, %B %e, %Y' }}. Check out [Join TTS Hiring Process]({{site.baseurl}}/hiring-process/) to learn more about the application process.
+  {% endif %}
 {% endif %}
 
-## Basic Information
+{% if page.state != 'upcoming' %}
+  {{ page.org }} is hiring for GS-{{ page.gs_level }} {{ page.title }}. This page contains information related to the role as well as a link to submit your application. Check out [Join TTS Hiring Process]({{site.baseurl}}/hiring-process/) to learn more about the application process.
+{% endif %}
 
 **Location:**
```

Comment:
@ctro The issue is that we have to match the required job announcement format, which requires the `Basic Information` to be where it is. This isn't a problem when it's just upcoming, which is why we put that info at the top. But once it's `open` and is an official job announcement, we have to adhere to the format. One thing we could do is that I've done here. Duplicate the `Location`. `Salary Range` and `Who May Apply` sections and always keep them at the top.

Reply:
:+1: OK, gotcha.  In that case I think duplicating these pieces makes sense.